R/bestlag.R

```diff
@@ -11,8 +11,11 @@
 #' @param C a numeric value of C used in computing weighted correlation
 #' @return a vector of best lags used in the dataset, one per gene
 #'
-#' @example best.lag(array(rnorm(20), c(4, 5)), timepoints = c(0, 5, 10,
-#'          20, 40), C = 300)
+#' @example
+#' best.lag(data = array(rnorm(20), c(4, 5)), timepoints = c(0, 5, 10, 20, 40), C = 300)
+#' best.lag(data = array(runif(100), c(5, 20)), timepoints = seq(2, 40, 2), C = 10)
+#' best.lag(data = array(runif(100), c(5, 20)), timepoints = seq(2, 40, 2), max.lag = 2, C = 10)
```

Comment:
This is not related to the documentation, but we should add error checking to make sure `max.lag` is not too small.  Is 1 the minimum reasonable value?

If we add error checking here, we can add it for the other functions that allow `max.lag` as a parameter as well.

Reply:
1 is the minimum right now.

Reply:
We decided that this is a separate issue.  See #32.